examples/saving_collection.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+
+sys.path.insert(0, os.getenv("BUILDTEST_ROOT"))
+
+from buildtest.modules.util import Module
+
+a = Module(["zlib"])
+# save module collection with name "zlib"
+a.save("zlib")
+# show module collection "zlib"
+a.describe("zlib")
```

Comment:
I think what we need (actually, what I need) is a very small bash script example that shows me how to set up a small local site with my own modules. And then provide instructions to run these commands. This would give me the entire picture:

 - being on a cluster with Lmod
 - creating a dummy module
 - interacting with buildtest to test it

Without any of the bullets above, a new user like myself won't have the full picture.